src/jit/lclvars.cpp

```diff
@@ -2732,10 +2732,10 @@ void Compiler::lvaUpdateClass(unsigned varNum, CORINFO_CLASS_HANDLE clsHnd, bool
         shouldUpdate = true;
     }
 
-    JITDUMP("\nlvaUpdateClass:%s Updating class for V%02u from (%p) %s%s to (%p) %s%s\n", varNum,
-            shouldUpdate ? "" : " NOT", dspPtr(varDsc->lvClassHnd), info.compCompHnd->getClassName(varDsc->lvClassHnd),
-            varDsc->lvClassIsExact ? " [exact]" : "", dspPtr(clsHnd), info.compCompHnd->getClassName(clsHnd),
-            isExact ? " [exact]" : "");
+    JITDUMP("\nlvaUpdateClass:%s Updating class for V%02u", shouldUpdate ? "" : " NOT", varNum);
+    JITDUMP(" from(%p) %%s", dspPtr(varDsc->lvClassHnd), info.compCompHnd->getClassName(varDsc->lvClassHnd),
+            varDsc->lvClassIsExact ? " [exact]" : "");
+    JITDUMP(" to(%p) %%s\n", dspPtr(clsHnd), info.compCompHnd->getClassName(clsHnd), isExact ? " [exact]" : "");
```

Comment:
Should be `%s%s` in both places.

Reply:
Oops, I had some weird cut-and-paste effects in VS. I blame that ;-)

Reply:
@AndyAyersMS - I think I've got it right now (and this time I actually looked at the jitdump output for the test that was failing - ``JIT\Methodical\flowgraph\dev10_bug679955\indexMinusOne\indexMinusOne.exe``